ci/steps/brew-bump.sh

```diff
@@ -26,7 +26,10 @@ main() {
   git merge upstream/master
 
   echo "Pushing changes to cdrci/homebrew-core fork on GitHub"
-  git push origin master
+  # Source: https://serverfault.com/a/912788
+  # shellcheck disable=SC2016,SC2028
+  echo '#!/bin/sh\nexec echo "$HOMEBREW_GITHUB_API_TOKEN"' >"$HOME"/.git-askpass.sh
+  GIT_ASKPASS="$HOME/.git-askpass.sh" git push https://cdrci@github.com/cdrci/homebrew-core.git --all
```

Comment:
Does this all look right?

Reply:
Oh I guess we'll need to update the bot username though.

Reply:
Oh good call! When I ran it locally it wasn't executable. 

Ah! I changed it locally when testing but not here. Good catch too!

Reply:
fixed!